reporting/impl/src/main/resources/reports/resources/css/tech-report-punchcard.css

```diff
@@ -34,6 +39,22 @@ tr.headersGroup td div {
     white-space: nowrap;
 }
 
+tr.headersGroup .sort_asc div:after {
+    content: "\f107";
+    font-family: "FontAwesome";
+    font-size: 10px;
```

Comment:
I know it's a personal taste thing but i think it looks better without `font-size: 10px` (so it will have the inherited one - 16pt) and `font-weight: bold` so that it's consistent with the column header bold style.